packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -114,18 +114,17 @@ module.exports = {
     loaders: [
       // Default loader: load all assets that are not handled
       // by other loaders with the url loader.
-      // Note: This list needs to be updated with every change of extensions
-      // the other loaders match.
-      // E.g., when adding a loader for a new supported file extension,
-      // we need to add the supported extension to this loader too.
-      // Add one new line in `exclude` for each loader.
       //
-      // "file" loader makes sure those assets get served by WebpackDevServer.
-      // When you `import` an asset, you get its (virtual) filename.
-      // In production, they would get copied to the `build` folder.
-      // "url" loader works like "file" loader except that it embeds assets
-      // smaller than specified limit in bytes as data URLs to avoid requests.
-      // A missing `test` is equivalent to a match.
+      // Note:
+      // The "url" loader exclude list needs to be updated with every change of extensions
+      // that other loaders match.
+      // For example, when adding a loader for a new supported file extension (such as .sass),
+      // you need to add the supported extension (/\.sass$/) to this loader too.
```

Comment:
I think this could be easier to consume, how about this?
```
** STOP **
This loader handles all assets unless explicitly excluded.
The `exclude` list *must* be updated with every change to loader extensions.
It should contain one line for each loader.

** ADDING NEW LOADERS **
When adding a new loader, you must add its `test` as a new entry in the `exclude` list for this loader.
If you do not add the entry, your new loader may not work if it introduced a new extension.

"file" loader makes sure those assets end up in the `build` folder.
-snip-
```